crates/nu-parser/src/parse_keywords.rs

```diff
@@ -3154,7 +3154,7 @@ pub fn parse_let(working_set: &mut StateWorkingSet, spans: &[Span]) -> Pipeline
     garbage_pipeline(working_set, spans)
 }
 
-pub fn parse_const(working_set: &mut StateWorkingSet, spans: &[Span]) -> Pipeline {
+pub fn parse_const(working_set: &mut StateWorkingSet, spans: &[Span]) -> (Pipeline, Option<Span>) {
```

Comment:
Maybe worth adding as part of the doccomment what the content of the `Option<Span>` is, otherwise those tuple return values are hard to deal with.